### ISSUE TITLE
A Generic Visualization Approach for Convolutional Neural Networks

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#108]
# TL;DR
They proposed an Attention-based evidence visualization method, L2-CAF. The method is based on a filter that eliminates all features other than the target class from the feature map, and it has the advantage of being able to handle multiple modes, although it only provides a single-mode signal from Softmax output. It can be implemented without any restriction of the base network structure.

![Original L2-CAF Softmax Gaussian](https://user-images.githubusercontent.com/24524018/93066743-52fc4780-f6b5-11ea-937b-6aa70bc3f419.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.09748

# Submission Dates(yyyy/mm/dd)
2020/07/19

# Authors and institutions
Ahmed Taha, Xitong Yang, Abhinav Shrivastava, and Larry Davis
- University of Maryland, College Park

# Methods

# Results

# Comments
